pkg/parse/updater.go

```diff
@@ -125,7 +127,9 @@ func (u *Updater) update(ctx context.Context, cache *cacheForCommit) status.Mult
 	// Apply the declared resources
 	if !cache.applied {
 		declaredObjs, _ := u.Resources.DeclaredObjects()
-		if err := u.apply(ctx, declaredObjs, cache.source.commit); err != nil {
+		klog.Infof("%v objects that were declared: %v", len(declaredObjs), core.GKNNs(declaredObjs))
```

Comment:
The applier will log declared objects. Do we need an extra log here?
If it is still needed, I would suggest making it a V(3) level log.

Reply:
No it's not necessary. It's left over from testing. I went ahead and removed it